Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+// TODO: make this scroll!
+class PlaceDetailsCardView: UIView {
+
+    // TODO: accessibility labels (and parent view)
+    // TODO: set line height on all text. http://stackoverflow.com/a/5513730
+    lazy var titleLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = Colors.detailsViewCardPrimaryText
+        view.font = Fonts.detailsViewTitleText
+        view.numberOfLines = 0
+        view.lineBreakMode = .byWordWrapping
+        return view
+    }()
+
+    lazy var categoryLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = Colors.detailsViewCardPrimaryText
+        view.font = Fonts.detailsViewCategoryText
+        return view
+    }()
+
+    // TODO: Set styling.
+    lazy var urlLabel: UILabel = {
+        let view = UILabel()
+        view.textColor = .blue
+        view.font = Fonts.detailsViewCategoryText // TODO
+        return view
+    }()
+
+    lazy var travelTimeView: PlaceDetailsIconInfoView = {
+        let view = PlaceDetailsIconInfoView() // TODO: icon depending on walking/driving
+        return view
+    }()
+
+    lazy var hoursView: PlaceDetailsIconInfoView = {
+        let view = PlaceDetailsIconInfoView()
+        view.iconView.image = UIImage(named: "icon_times")
+        return view
+    }()
+
+    lazy var yelpReviewView: ReviewContainerView = {
+        let view = ReviewContainerView(color: Colors.yelp, mode: .detailsView)
```

Comment:
If things like font, color and padding are changing between details and carousel views, I would rather those values became variables on the `ReviewContainerView` and were set explicitly inside the `UIViewController`. If a view is to be used in more than one way, I would feel happier if it didn't encapsulate that knowledge inside itself, but each `UIViewController` encapsulated the knowledge about how it needs it's views configured.